Client/Frontend/Browser/TabManager.swift

```diff
@@ -852,7 +852,7 @@ extension TabManager {
                     self.restoreTabsInternal()
                 },
                 catch: { exception in
-                    Sentry.shared.send(message: "Failed to restore tabs: ", tag: SentryTag.tabManager, severity: .error, description: "\(exception ??? "nil")")
+                    log.error("Failed to restore tabs: \(String(describing: exception))")
```

Comment:
`exception.description` works too, right?

Reply:
I think it'll print the same thing, but I'll swap them anyways

Reply:
Yes, probably would, just looks ugly mostly, I think this is much better (albeit I find the other method's warnings dumb).